cmd/client/main.go

```diff
@@ -119,18 +121,20 @@ func withArgs(args []string, outf outFn, errorf errorFn) {
 	rootCmd.AddCommand(reportCmd(rootArgs, outf, errorf))
 	rootCmd.AddCommand(quotaCmd(rootArgs, outf, errorf))
 
-	if err := rootCmd.Execute(); err != nil {
-		errorf(err.Error())
-	}
+	return rootCmd
 }
 
 func main() {
-	withArgs(os.Args[1:],
+	rootCmd := GetRootCmd(os.Args[1:],
 		func(format string, a ...interface{}) {
 			fmt.Printf(format, a...)
 		},
 		func(format string, a ...interface{}) {
-			fmt.Fprintf(os.Stderr, format, a...)
-			os.Exit(1)
+			fmt.Fprintf(os.Stderr, format+"\n", a...)
+			os.Exit(-1)
```

Comment:
ok. That's reasonable. I'm confused by the switch from 1 to -1 (where 1 seems to be a fairly standard catch-all for general errors and is the exit code used by log.Fatal).